controllers/routemap.go

```diff
@@ -32,7 +33,8 @@ type routemap struct {
 
 type version struct {
 	Resources []resource `json:"resources,omitempty"`
-	Weight    *uint32    `json:"weight,omitempty"`
+	Weight    *uint32    `json:"label,omitempty"`
```

Comment:
This seems like a bug? `json:"weight,...`

Reply:
Oh, that is strange. I think this may have come from when I was rebasing.